### PR TITLE
Added yarn and node.js instruction specifically for debian/ubuntu

### PR DESCRIPTION
**Pull request by aistosi**
Added some additional instruction for installation of Calypso, particularly the issues I encountered during installation not documented anywhere. First PR, go easy on me :)

### CODE REVIEW

File: docs/install.md [hunk 1]
```diff
@@ -14,8 +14,10 @@ You can install Calypso directly on your machine by following the next steps, or
...
+-	Install the [Node.js](http://nodejs.org/) and matching [NPM](https://www.npmjs.com/) [versions](https://nodejs.org/en/download/releases/) listed in the engines section of [package.json](https://github.com/Automattic/wp-calypso/blob/master/package.json). On Mac OS X and Linux using [nvm](https://github.com/creationix/nvm) or [n](https://github.com/tj/n) makes it easy to manage `node` versions. On Windows you may want to try [nvm-windows](https://github.com/coreybutler/nvm-windows) or [nodist](https://github.com/marcelklehr/nodist). 
+-	Please note that in Debian/Ubuntu versions of Linux, `node` command is renamed to `nodejs`. This will cause Calypso to fail during installation. Follow the instructions [here](https://stackoverflow.com/a/18130296) to create a symlink to `node`.
 -	[Git](http://git-scm.com/). Try the `git` command from your terminal, if it's not found then use this [installer](http://git-scm.com/download/).
+-	[Yarn](https://yarnpkg.com). Type the `yarn` command in your terminal. In case it is not found, follow the instructions [here](https://yarnpkg.com/lang/en/docs/install/) to install it on your system.
```
Comment (spen) [inline docs/install.md]: I'm not sure of the status of our yarn usage so I'll hold off hitting accept just yet - @gwwar what are your thoughts on giving advice for using yarn in the install docs?

Comment (gwwar) [reply docs/install.md]: We likely won't be adopting yarn due to npm 5 including a lot of the features folks like. (We can try updating node/npm in Oct).

So it's not recommended for development. If it happens to work currently, that's great but we won't be checking in on related bugs from differences in npm vs yarn.

Comment (aistosi) [reply docs/install.md]: @spen @gwwar  During the install on Ubuntu, the install in the dashboard crashed with message that yarn was not installed. Do you guys need to remove the instructions or keep them instead? Maybe, move this to somewhere else?

Comment (spen) [reply docs/install.md]: @gwwar - I remember seeing this point the last time I started thinking about yarn, but I'm oft forgetful :P Thanks for the reminder!

>  During the install on Ubuntu, the install in the dashboard crashed

@aistosi - Are you seeing this issue with the current install guides (before the additions above)?

Comment (aistosi) [reply docs/install.md]: @spen Yeah, I was following the readme.md and install.md and was installing Calypso over the fresh install of Ubuntu. Not sure if there are any other guides available relating to installation of Calypso.

Comment (gwwar) [reply docs/install.md]: cc @DanReyLop do you think the node version check is misbehaving here?

**Comment by aistosi** [comment]
Not sure why test failed. Something about eslint?

File: docs/install.md (hunk 1 above)
Comment (DanReyLop) [reply docs/install.md]: > cc @DanReyLop do you think the node version check is misbehaving here?

It looks like it, but I don't see how it's possible. Looking at [this line](https://github.com/Automattic/wp-calypso/blob/master/package.json#L222), the check is:
`... && ( check-node-version --package || exit 0 )`
Which means that, worst case scenario, it should spit a warning, but never abort the build.

@aistosi Maybe you saw the warning in the console, but the real crash was caused by something else afterwards? `npm` is very verbose, so the answer is probably there. What were the last lines of the output?

**Comment by gwwar** [comment]
> would it be ok to keep the first part related to Debian/Ubuntu if we don't need Yarn instructions atm?

sounds good to me 👍

**Comment by gwwar** [comment]
@vindl if @aistosi doesn't follow up, feel free to land this, then create a quick PR to update docs.

**Comment by vindl** [comment]
@gwwar merged and created a follow up PR as suggested: https://github.com/Automattic/wp-calypso/pull/20084.

**Pull request merged**